src/pages/LogIn/Login.tsx

```diff
@@ -19,10 +20,11 @@ class LogInPage extends React.Component<AllProps> {
         //footerListVariants="inline"
         brandImgSrc={brandImg}
         brandImgAlt="PatternFly logo"
```

Comment:
Hey, could you do me a favour and change the alternative text to `DarwinAI logo`? Thanks

Reply:
Done!